.github/CODEOWNERS

```diff
@@ -1,7 +1,12 @@
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# these owners will be requested for review when someone
-# opens a pull request.
-*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
+* @dav3r @jsf9k
+
+# Let jsf9k own the sometimes-touchy AWS and Python playbooks, as well
+# as the packer.json file.
+/src/aws.yml @jsf9k
+/src/packer.json @jsf9k
```

Comment:
FYI, I'm fine if you add me as an owner on the `packer.json` file in this and most other repos.